<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QMainWindow" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pinnacle</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabs">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>401</width>
      <height>251</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="buttons_tab">
     <attribute name="title">
      <string>Sales Performance Of Vehicle Production</string>
     </attribute>
     <widget class="QPushButton" name="employees_button">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>120</y>
        <width>281</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Employees's performance</string>
      </property>
     </widget>
     <widget class="QPushButton" name="productLines_button">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>170</y>
        <width>281</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Productlines' performance</string>
      </property>
     </widget>
     <widget class="QPushButton" name="etl_wh_button">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>281</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Perform ETL for Warehouse</string>
      </property>
     </widget>
     <widget class="QPushButton" name="etl_db_button">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>281</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Initialize Operational Database</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
